ktlint-ruleset-standard/src/main/kotlin/com/pinterest/ktlint/ruleset/standard/NoMultipleSpacesRule.kt

```diff
@@ -18,12 +18,21 @@ class NoMultipleSpacesRule : Rule("no-multi-spaces") {
             // allow multiple spaces in KDoc in case of KDOC_TAG for alignment, e.g.
             // @param foo      stuff
             // @param foobar   stuff2
-            !(node.treePrev.elementType == KDOC_MARKDOWN_LINK && node.treeParent.elementType == KDOC_TAG)
```

Comment:
You could've simply put a `?` after `treePrev` and `treeParent` instead of introducing a new method :)

Reply:
Agreed, I've updated the code in ef5130b30609ab4701da47f3fc2b87b386ee9adc